src/pages/Sessions.tsx

```diff
@@ -201,8 +256,34 @@ const Sessions = () => {
         </Box>
         {selectedSession && (
           <>
+            <FormControl variant="outlined">
+              <InputLabel id="view">View</InputLabel>
+              <Select
+                id="select"
+                label="view"
+                labelId="view"
+                value={!attendanceView ? "default" : "attendance"}
+              >
+                <MenuItem
+                  value="default"
+                  onClick={() => setAttendanceView(false)}
+                >
+                  Default view
+                </MenuItem>
+                <MenuItem
+                  value="attendance"
+                  onClick={() => setAttendanceView(true)}
+                >
+                  Attendance view
+                </MenuItem>
+              </Select>
+            </FormControl>
             <Box flexShrink={0}>
-              <Button variant="outlined" onClick={handleOpenFormDialog}>
+              <Button
+                variant="outlined"
+                onClick={handleOpenFormDialog}
+                style={{ marginLeft: "20px", height: "56px" }}
```

Comment:
can we use makeStyles instead of inline styling? there should be a few examples across the app

Reply:
i think this button also got stretched vertically somehow - can we restore the original height?

before:
<img width="1148" alt="Screen Shot 2021-08-07 at 5 09 37 PM" src="https://user-images.githubusercontent.com/37346399/128613897-6f7a9157-141c-467a-ba00-6b272c5d0931.png">

after:
<img width="1146" alt="Screen Shot 2021-08-07 at 5 09 16 PM" src="https://user-images.githubusercontent.com/37346399/128613891-48bed3a3-d305-4cf3-aff2-ab5f26d4fcc5.png">